lib/matplotlib/backends/backend_mixed.py

```diff
@@ -68,6 +69,71 @@ def __getattr__(self, attr):
 
+    # need to wrap each drawing function that might be called on the rasterized
+    # version of the renderer to save what the "true" bbox is for scaling the
+    # output correctly
+    # the functions we might want to overwrite are:
+    # `draw_path`, `draw_image`, `draw_gouraud_triangle`, `draw_text`,
+    # `draw_markers`, `draw_path_collection`, `draw_quad_mesh`
+
+    def _update_true_bbox(self, bbox, transform=None):
+        """Convert to real units and update"""
+        if transform is None:
+            transform = IdentityTransform()
+        bbox = bbox.transformed(transform + Affine2D().scale(
+            self._figdpi / self.dpi))
+        if self._true_bbox is None:
+            self._true_bbox = bbox
+        else:
+            self._true_bbox = Bbox.union([self._true_bbox, bbox])
+
+    def draw_path(self, gc, path, transform, rgbFace=None):
+        if self._rasterizing > 0:
+            bbox = Bbox.null()
+            bbox.update_from_path(path, ignore=True)
+            self._update_true_bbox(bbox, transform)
+        return self._renderer.draw_path(gc, path, transform, rgbFace)
+
+    def draw_path_collection(self, gc, master_transform, paths, all_transforms,
+                             offsets, offsetTrans, facecolors, edgecolors,
+                             linewidths, linestyles, antialiaseds, urls,
+                             offset_position):
+        if self._rasterizing > 0:
+            bbox = Bbox.null()
+            # TODO probably faster to merge all coordinates from path using
+            # numpy for large lists of paths, such as the one produced by the
+            # test case tests/test_backed_pgf.py:test_mixed_mode
+            for path in paths:
+                bbox.update_from_path(path, ignore=False)
+            self._update_true_bbox(bbox, master_transform)
+        return self._renderer.draw_path_collection(
+                gc, master_transform, paths, all_transforms, offsets,
+                offsetTrans, facecolors, edgecolors, linewidths, linestyles,
+                antialiaseds, urls, offset_position)
+
+    def draw_quad_mesh(self, gc, master_transform, meshWidth, meshHeight,
+                       coordinates, offsets, offsetTrans, facecolors,
+                       antialiased, edgecolors):
+        if self._rasterizing > 0:
+            # TODO should check if this is always Bbox.unit for efficiency
+            bbox = Bbox.null()
+            cshape = coordinates.shape
+            flat_coords = coordinates.reshape((cshape[0]*cshape[1], cshape[2]))
+            bbox.update_from_data_xy(flat_coords, ignore=True)
+            self._update_true_bbox(bbox, master_transform)
+
+        return self._renderer.draw_quad_mesh(
+                gc, master_transform, meshWidth, meshHeight, coordinates,
+                offsets, offsetTrans, facecolors, antialiased, edgecolors)
+
+    def draw_gouraud_triangle(self, gc, points, colors, transform):
```

Comment:
This should probably be `draw_gouraud_triangles`:
https://github.com/matplotlib/matplotlib/blob/d77801ea94b06b990fc2b535a42522b45143e630/lib/matplotlib/backend_bases.py#L291-L292

https://matplotlib.org/stable/api/prev_api_changes/api_changes_3.7.0.html#draw-gouraud-triangle